clients/vscode-hlasmplugin/src/test/syntaxes/endevor/parenthesis.listing.snap

```diff
@@ -163,8 +163,8 @@
 #                                                                       ^^^ source.hlasmListing punctuation.definition.tag.hlasm
 #                                                                          ^ source.hlasmListing
 #                                                                           ^^ source.hlasmListing
```

Comment:
Same thing. The (single) space before `REMARK(REM)` should be part of the comment.